torch/_export/serde/serialize.py

```diff
@@ -1626,11 +1626,13 @@ def save_graph_module(self) -> Iterator[None]:
             self.module,
             self.serialized_name_to_node,
             self.serialized_name_to_meta,
+            self.unbacked_symbols
```

Comment:
We need stash unbacked_symbols before deserialize_subgraph because the subgraph may have different set of the unbacked_symbols.